### ISSUE TITLE
Update failed even after turning off firewall, antivirus and proxy

### ISSUE THREAD

**Issue by AsianSpaghetti** [sstokic-tgm/JFTSE#153]
I get the following message when clicking "Launch" even after disabling my firewall, my antivirus, and proxy as the Discord FAQ page suggested, how can I fix it?

![image](https://user-images.githubusercontent.com/102522018/213011065-cdc60f72-78ab-4d91-9963-68781d1f8747.png)

Thank you in advance for your support!

**Comment by AsianSpaghetti** [comment]
Hi,

thank you for answering, after opening this issue I scrolled through the Discord server hoping to find someone else with my same problem, I've read on the Discord to ask for the Mega Files through ModMail and immediately did so but I don't know if it went through, do I need to do anything else after sending a message and giving the bot permissions?

I created a folder called "Fantasy Tennis" containing the launcher and config files on my Desktop before launching and got this error, let me know if there's any way I can fix this without bothering further I'm just so excited to have found my childhood game again, I can't wait to play.

Thanks in advance!
